Npgsql.sln

```diff
@@ -29,6 +29,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Npgsql.GeoJSON", "src\Npgsq
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Npgsql.Specification.Tests", "test\Npgsql.Specification.Tests\Npgsql.Specification.Tests.csproj", "{A77E5FAF-D775-4AB4-8846-8965C2104E60}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{87B8241E-FFEC-4D4D-A867-9526261ACAD8}"
+ProjectSection(SolutionItems) = preProject
+	Directory.Build.props = Directory.Build.props
+	Directory.Build.targets = Directory.Build.targets
+EndProjectSection
+EndProject
```

Comment:
Let's revert this because solution items is a legacy feature, all modern IDEs able to show a folder content. When some change happens with files mentioned in the solution, there is no reflection in IDE. Even now you forgotten to add the same to `Npgsql.all.sln`.

Reply:
Why do you say it's a legacy feature? Any reason not to allow easy access to these important build files without switching to folder content view (where there's lots of other irrelevant files)?

Reply:
Solutions file format is a legacy by itself (: As I said previously we removed them because of constantly forgetting to update them. Now you forgot to change the second solution file too which shows how often this happens.

Reply:
Hello guys.. .. i am new about postgresql i want to use postgresql using my vs 2010 c# and i dont know how will i connect it and looking for the npgsql.dll but no luck  i found the source here but i did not found any dll file to be added on my reference on visual studio 2010 c#

Thank you

Reply:
> Solutions file format is a legacy by itself (:

Yes, like an ugly house you inherited. You don't like it but it's still the place where many people live in.